### ISSUE TITLE
Limit numbers of members per page

### ISSUE THREAD

**Issue by bonald** [dec0dOS/zero-ui#18]
We have more than 1000 members and when loading the network page it loads every members making the browser unresponsive.

**Comment by dec0dOS** [comment]
Hello, @bonald! Thanks for reaching out.
Implementing such an additional API would require some effort, because the original ZeroTier Central API does not have such an option, and breaking the complitability is not the option.

**Comment by bonald** [comment]
maybe its laggy because your also outputting to browser console?

**Comment by dec0dOS** [comment]
I don't think so. You could disable the log function and try another browser. Currently, I think the problem is with the React rendering. It may require some time to investigate.
